config/kubernetes/org.yaml

```diff
@@ -661,6 +663,7 @@ members:
 - sjpotter
 - sjug
 - SlickNik
+- smana
```

Comment:
nit: peribolos doesn't really care about capitalization in usernames, but can we fix this here to `Smana` since that's the actual username to remain consistent with others?

Reply:
done!